fmriprep/workflows/fieldmap/phdiff.py

```diff
@@ -73,7 +73,8 @@ def _pick1st(inlist):
         return inlist[0]
 
     # Read phasediff echo times
-    meta = pe.Node(ReadSidecarJSON(), name='meta', mem_gb=0.01, run_without_submitting=True)
+    meta = pe.Node(ReadSidecarJSON(bids_validate=False), name='meta',
+                   mem_gb=0.01, run_without_submitting=True)
```

Comment:
For nodes that construct BIDS layouts, we may want to consider whether to use `run_without_submitting`. Large datasets can take a long time, and `run_without_submitting` means we block the master thread.

Maybe we can time the first creation, and if it's >2sec (or something) go ahead and submit?

Reply:
Let's submit it, I feel like timing the first creation is somewhat splitting hairs.